corehq/apps/domain/tests/test_domain_transfer.py

```diff
@@ -184,7 +184,8 @@ def test_permissions_for_transfer_domain_view(self):
         # No one logged in
         resp = self.client.get(reverse('transfer_domain_view',
                                        args=[self.domain.name]), follow=True)
-        self.assertEqual(resp.status_code, 404)
+        self.assertEqual(resp.template_name, 'login_and_password/login.html',
```

Comment:
Basically if someone changes the template_name or directory name for this template, this would break. Since you want to make sure that it's redirected to login URL, you can compare the URL location of `resp` and login url.
Alternatively, you can remove follow=True (which follows the redirected URL) and do following `self.assertRedirects(resp, login_url)`, which checks for redirect code and the expected redirected URL.